backend/src/main/java/gov/cdc/usds/simplereport/service/DataHubClient.java

```diff
@@ -0,0 +1,20 @@
+package gov.cdc.usds.simplereport.service;
+
+import feign.Param;
+import gov.cdc.usds.simplereport.config.DataHubClientConfiguration;
+import gov.cdc.usds.simplereport.service.model.reportstream.UploadResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@FeignClient(
+    name = "datahub",
+    url = "${datahub.url}",
```

Comment:
whats our strategy to specify this in prod?

Reply:
My understanding is @nathancrtr's infra PR will specify the correct url.

Reply:
@zdeveloper Dan's correct. The rundown is that the relevant `datahub-url-${env}` and `datahub-api-key-${env}` _currently_ exist in the Key Vault, and the https://github.com/CDCgov/prime-simplereport/pull/3800 PR adds a little Terraform to pull those secrets in as envvars so we can read them from code.